package.json

```diff
@@ -28,13 +29,14 @@
     "@lido-sdk/helpers": "^1.4.3",
     "@lido-sdk/react": "^1.18.1",
     "@lido-sdk/web3-react": "^1.26.2",
-    "@lidofinance/lido-ui": "^2.16.2",
```

Comment:
I wonder why `3.0.0-next.16` instead of `3.16.0` or something like that.

Reply:
this is very good point. @solidovic can you please release `3.0.0` before merge and update the version?

Reply:
ok.